app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -156,6 +159,11 @@ private String getSessionToken() {
         return session[0];
     }
 
+    ClientInfo getClientInfoFromUserAgentHeader() {
+    	return ClientInfo.fromUserAgentCache(
+			request().getHeader(BridgeConstants.USER_AGENT_HEADER));
+    }
```

Comment:
Is it worth logging this, just to see that it's correctly parsing UA strings in the real world? (Maybe filter the logging to non-production, if we're concerned about log volume.)